src/features/VisualNovel/pages/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { FC, Fragment, useEffect, useState } from 'react';
+import { Heading, Link } from '@chakra-ui/react';
+
+import languageCodes from 'iso-639-1';
+import { useParams } from 'react-router';
+import cls from './OverviewPage.module.css';
+import { StaffRoles, STAFF_ROLES } from '../../../../utils/types/staffRoles';
+import { VisualNovelLinks } from '../../../../utils/types/visualNovelLinks';
+import { TagBlock } from '../../components/TagBlock/TagBlock';
+import { CharacterCard } from '../../components/CharacterCard/CharacterCard';
+import { useVisualNovelQuery, useCharactersQuery, useReleasesQuery, useTagsQuery } from '../../queries';
+import { Release } from '../../../../models/release';
+
+/**
+ * Overview tab page.
+ */
+export const OverviewPage: FC = () => {
+  const { id } = useParams();
+  const [developers, setDevelopers] = useState<string[]>([]);
+  const [publishers, setPublishers] = useState<Record<string, string[]>>(
```

Comment:
Actually, as I see, `developers` and `publishers` are not state of the component, they are just dependable values that are calculated from `releases` data. Just create `const` variables from `fillDeveloperTeam` function. I think it should do the job.